### ISSUE TITLE
TOUCHSCREEN INSTALLATION QUERIES

### ISSUE THREAD

**Issue by raynessupreme** [pideck/pideck#147]
I'm back again, its been a minute...

I've revisited PiDeck after gathering the majority of parts and free time for a long-awaited project.

I have a Waveshare Spotpear 3.5" LCD Touchscreen that is powered via the Pi GPIO Pins and has an HDMI coupler to support the video output on the screen.

I power it on and I get a "no signal" response, which is a positive considering this thing has been sat in a drawer for several years, though in order to activate the screen, I have to install a script from Github via an Internet connection.

Here is where I get stuck...

Being a Linux n00b, I have gotten as far as SSHing in to the Pi from PuTTY via an ethernet connection and not much else. I follow tutorial documentation on how to gain an internet connection on your Pi, but whatever terminal commands I use, the Pi does not respond to them. I shall keep attempting things, though I fear that my lack of understanding and frustation in not having tutorial commands work may lead me to shelve the project again like I did some time ago.

I have also attempted to use WINScp to get in to the inner workings of the Pi and copy over a Tar.gz file of the Touchscreen Installer, though I cannot gain access to the Root file and nor can I run the installer script from home/pi (where the SCP defaults to). I have attempted to copy the files from home/pi to the root, but the system does not let me.

How have others gotten their Touchscreens Installed with PiDeck? I have always assumed that PiDeck is just xWax running over Raspbian, so I figured I can get in to the Debian OS somehow to configure my SSID and whatnot. I've even used the fancy new RPi Image Flashing tool that gives you the option of setting up your Network alongside flashing the OS image, but this did not appear to work either, considering I had to revert back to Ethernet instead of WiFI.

Any support is gratefully appreciated. I just want to get the screen configured, then I can crack on with the rest of my project.

Many Thanks,

Shaun.

**Comment by raynessupreme** [comment]
After some further digging, it turned out my Touchscreen only required some additional text adding to the Config file of a fresh PiDeck Grandmaster installation. I have now closed this issue. Thank you :)

**Comment by danielhjames** [comment]
Hi Shaun, thanks for the update. Could you provide a link to the config you used as a guide for other users of this particular touchscreen please?

**Comment by raynessupreme** [comment]
Hi Dan.

Yes, of course.

Section 4 of the Software Setting part of this Wiki Page helped me to add in the Screen. This worked great as Touch functionality appeared to be enbedded as default.

I then used the No KMS or FKMS driver loaded Section of Screen Orientation Settings part to rotate the Screen by 180 degrees, though I have noticed that the Touch functionality has ceased to work, or works tempramentally. I am wondering whether the PiDeck buttons are still in the same location despite the screen rotation. If I continue to run in to issues, I may just remove the text from the Config file.

https://www.waveshare.com/wiki/3.5inch_HDMI_LCD
